src/main/scala/it/pagopa/interop/attributeregistryprocess/common/readmodel/ReadModelQueries.scala

```diff
@@ -0,0 +1,47 @@
+package it.pagopa.interop.attributeregistryprocess.common.readmodel
+
+import it.pagopa.interop.attributeregistrymanagement.client.model.Attribute
+import it.pagopa.interop.attributeregistryprocess.api.impl.attributeManagementFormat
+import it.pagopa.interop.commons.cqrs.service.ReadModelService
+import org.mongodb.scala.Document
+import org.mongodb.scala.model.Filters
+import org.mongodb.scala.model.Aggregates.{`match`, count, project, sort}
+import org.mongodb.scala.model.Projections.{computed, fields, include}
+import org.mongodb.scala.model.Sorts.ascending
+
+import scala.concurrent.{ExecutionContext, Future}
+
+object ReadModelQueries {
+  def getAttributes(kinds: List[String], offset: Int, limit: Int)(
+    readModel: ReadModelService
+  )(implicit ec: ExecutionContext): Future[PaginatedResult[Attribute]] = {
```

Comment:
The stored model is the Persistent, not the Api
```suggestion
  )(implicit ec: ExecutionContext): Future[PaginatedResult[PersistentAttribute]] = {
```
You can import its codecs from `it.pagopa.interop.attributeregistrymanagement.model.persistence.JsonFormats._`